helmctlr/helm.go

```diff
@@ -151,6 +169,46 @@ func (c Controller) marshallCR(r *unstructured.Unstructured) ([]byte, error) {
 	return yaml.Marshal(re)
 }
 
+func (c Controller) setReleaseConfigMapOwnerReferences(r *unstructured.Unstructured) error {
+	rlsName := c.releaseName(r)
+	selector, err := metav1.LabelSelectorAsSelector(&metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"OWNER": "TILLER",
+			"NAME":  rlsName,
+		},
+	})
+	if err != nil {
+		return err
+	}
+	opts := metav1.ListOptions{
+		LabelSelector: selector.String(),
+	}
+	helmReleaseConfigMaps, err := c.KubeClient.CoreV1().ConfigMaps(c.Namespace).List(opts)
+	if err != nil {
+		return err
+	}
+
+	for _, releaseCM := range helmReleaseConfigMaps.Items {
+		obj, err := meta.Accessor(&releaseCM)
+		if err != nil {
+			return err
+		}
+		if metav1.IsControlledBy(obj, r) {
+			continue
+		}
+		controllerRef := metav1.NewControllerRef(r, r.GroupVersionKind())
+		ownerRefs := []metav1.OwnerReference{
```

Comment:
do we need to be careful here? shouldn't we pull existing ownerrefs from the configmap first and only add to them (unless it's to the same CR group/version/kind, then overwrite?

Reply:
Tiller currently doesn't set owner references, so I'm pretty sure we're fine, but I could try to support that.

Reply:
Since we know tiller isn't currently using ownerRefs configmaps, and we're going to be prescribing how to run tiller/what version, can we punt on this?